Should not require VCF's to have `AF`

```
bash ./annotation-tools/annotation_suite_wrapper.sh -i=./ -o=./mafs/ -m=./mafs/final.maf -c=FOO -s=WEX -p=./annotation-tools/
	INPUT_DATA_DIRECTORY=./
	OUTPUT_DATA_DIRECTORY=./mafs/
	MERGED_MUTATION_FILENAME=./mafs/final.maf
	CENTER_NAME=FOO
	SEQUENCE_SOURCE=WEX
	ANNOTATION_SUITE_SCRIPTS_HOME=./annotation-tools/
rm: cannot remove './mafs//annotated': Is a directory
rm: cannot remove './mafs//processed': Is a directory
	[INFO] standardizeMutationFilesFromDirectory(), standardized mutation files from ./ will be written to ./mafs//processed
Loading data from input directory: ./
	Searching for files with extensions: vcf, maf, txt
Loading data from file: ./Strelka_JHU2084-PN_variants_VEP.ann.vcf
Traceback (most recent call last):
  File "./annotation-tools//standardize_mutation_data.py", line 1624, in <module>
    main()
  File "./annotation-tools//standardize_mutation_data.py", line 1620, in main
    generate_maf_from_input_data(input_directory, output_directory, extensions_list, center_name, sequence_source)
  File "./annotation-tools//standardize_mutation_data.py", line 1584, in generate_maf_from_input_data
    maf_data = extract_vcf_data_from_file(input_filename, center_name, sequence_source)
  File "./annotation-tools//standardize_mutation_data.py", line 1403, in extract_vcf_data_from_file
    maf_record = create_maf_record_from_vcf(sample_id, center_name, sequence_source, vcf_data, is_germline_data, matched_normal_sample_id, tumor_sample_data_col)
  File "./annotation-tools//standardize_mutation_data.py", line 1364, in create_maf_record_from_vcf
    resolve_vcf_counts_data(vcf_data, maf_data, matched_normal_sample_id, tumor_sample_data_col)
  File "./annotation-tools//standardize_mutation_data.py", line 1259, in resolve_vcf_counts_data
    (t_ref_count, t_alt_count, t_depth) = resolve_vcf_allele_depth_values(tumor_sample_format_data, vcf_alleles, variant_allele_idx, vcf_data)
  File "./annotation-tools//standardize_mutation_data.py", line 1235, in resolve_vcf_allele_depth_values
    not is_missing_vcf_data_value(depth) and not is_missing_vcf_data_value(mapped_sample_format_data["AF"])
KeyError: 'AF'
[ERROR] standardizeMutationFilesFromDirectory(), error encountered while running ./annotation-tools//standardize_mutation_data.py
```

--- a/standardize_mutation_data.py
+++ b/standardize_mutation_data.py
@@ -1232,7 +1232,7 @@ def resolve_vcf_allele_depth_values(mapped_sample_format_data, vcf_alleles, vari
     # if depth has been resolved but not ref_count, alt_count then calculate the counts
     # if allele frequency vcf field "AF" exists
     if (
-        not is_missing_vcf_data_value(depth) and not is_missing_vcf_data_value(mapped_sample_format_data["AF"])
+        not is_missing_vcf_data_value(depth) and ("AF" in mapped_sample_format_data and not is_missing_vcf_data_value(mapped_sample_format_data["AF"]))
         and (is_missing_vcf_data_value(ref_count) or is_missing_vcf_data_value(alt_count))
         ):
         # check if ref count or alt count are still missing but AF VCF field is available

